src/testSession.ts

```diff
@@ -102,9 +111,14 @@ export class TestSession {
     // Write the test session options used to create this session
     fsCore.writeJsonSync(path.join(this.dir, 'testSessionOptions.json'), JSON.parse(JSON.stringify(options)));
 
+    // have to grab this before we change the home
+    transferExistingAuthToEnv();
+
     // Set the homedir used by this test, on the TestSession and the process
+    // TODO: does this work on Windows?
```

Comment:
It should be USERPROFILE, see https://nodejs.org/api/os.html#os_os_homedir - but if you keep the process.env.HOME check above rather than just relying on os.homedir() then it probably doesn't matter here.